server/test/testLogin.js

```diff
@@ -0,0 +1,21 @@
+// const chai = require('chai');
```

Comment:
Do the tests need to be commented out for Travis CI? I see one test commented out and the other not commented. 

I'm only checking for consistency.